cli/kubectl_kadalu/storage_add.py

```diff
@@ -92,16 +93,13 @@ def storage_add_validation(args):
                    (1 if args.external is not None else 0)
 
     if num_storages == 0:
-        print("Please specify atleast one storage", file=sys.stderr)
+        print("Please specify at least one storage", file=sys.stderr)
         sys.exit(1)
 
     storage_mismatch = False
-    if (args.type == "Replica1" and num_storages != 1) or (
-            args.type == "Replica3" and num_storages != 3
-    ):
-        storage_mismatch = True
-
-    if args.type == "Replica2" and num_storages != 2:
+    if (args.type == "Replica1" and num_storages != 1) or
+       (args.type == "Replica2" and num_storages != 2) or
+       (args.type == "Replica3" and num_storages != 3):
```

Comment:
I did the ugly coding due to some pylint warning saying more than 5 checks in a single if case is not 'advised'. But this looks clean while reading the code. I prefer this.

Two things:

`# pylint: disable=too-many-boolean-expressions` (R0916)  Above this if case.

remove the usage of `storage_mismatch` and merge the below if condition too.

Otherwise, it looks good.